src/lib/int.test.ts

```diff
@@ -2075,24 +2068,24 @@ describe('int', () => {
 
       describe('from() ', () => {
         describe('fromNumber()', () => {
-          it('should be the same as Field(1)', () => {
-            const x = UInt32.fromNumber(1);
-            expect(x.value).toEqual(new UInt32(Field(1)).value);
+          it('should be the same as Field.one', () => {
+            const x = UInt32.from(1);
+            expect(x.value).toEqual(new UInt32(Field.one).value);
```

Comment:
isn't this change and similar ones going against our deprecation of `Field.one`? 😅

Reply:
Oh ye thats right, I accidentally kept the old changes from the merge conflict